cmd/whisperImporter/whisperReader/main.go

```diff
@@ -0,0 +1,298 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/kisielk/whisper-go/whisper"
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/cmd/whisperImporter"
+	"github.com/raintank/metrictank/mdata/chunk"
+	"gopkg.in/raintank/schema.v1"
+)
+
+var (
+	whisperDirectory = flag.String(
+		"whisper-directory",
+		"/opt/graphite/storage/whisper",
+		"The directory that contains the whisper file structure",
+	)
+	chunkSpanStr = flag.String(
+		"chunkspan",
+		"10min",
+		"duration of chunks",
```

Comment:
I'm kind of unsure what's the best way to implement that. My first idea would be this:

I accept a parameter that allows a list of chunk spans to be specified. Every time a whisper file gets converted into chunks the first archive (raw) will get the first chunk span of the list applied to it, if there are more archives in the whisper file the second gets the second chunk span etc. 
If somebody has a bunch of whisper files and wants to apply different chunk spans to them they'll just have to run the utility multiple times, otherwise that would get too complicated.